resources/views/projects/create.blade.php

```diff
@@ -10,37 +10,28 @@
 		text-align: center;
 		color: #C7C7C7;
 	}
-</style>
-<div class="padded content">
-
-	@if($errors->any())
-		<div class="ui error message">
-			<ul>
-				@foreach ($errors->all() as $error)
-					<li>{{ $error }}</li>
-				@endforeach
-			</ul>
-		</div>
-	@endif
 
+	.imgUploader {
+		margin-bottom: 20px;
+	}
 
-	<form class="ui form" method="POST" action="/projects">
+</style>
+<div class="padded content">
+	<form class="ui form {{ $errors->any() ? 'error': '' }}" method="POST" action="/projects">
 		@csrf
 		<!-- Project Image -->
-		<div class="projectImage field">
+		<div class="projectImage field {{ $errors->has('projectImage') ? 'error': '' }}">
 			<label for="projectImage">Project Image</label>
 			<div class="projectImageUploaderContainer">
 				<div class="imgUploader" class="imagePreviewContainer">
 					<img src="https://lorempixel.com/400/400" alt="Project Image" class="ui medium image" id="projectImagePreview"/>
 				</div>
-				<a href="#" class="imgUploader">
-					<button class="upload-image-button ui button primary" type="button">Upload Image</button>
-				</a>
+				<button class="imgUploader upload-image-button ui button primary" type="button" href="#">Upload Image</button>
 			</div>
 			<input id="imgInput" name="projectImage" type="file" style="display:none" accept="image/x-png,image/jpeg,image/png" onchange="updateImage(this)">
 		</div>
 		<!-- Project Name -->
-		<div class="field">
+		<div class="field {{ $errors->has('projectName') ? 'error': '' }}">
 			<label for="projectName">Project Name</label>
 			<input type="text" name="projectName" placeholder="Project Name">
```

Comment:
For all inputs, let's name sure we follow the conventions:

1. Capitalize only the first letter of the first word in the label `Project name`
2. Placeholders should include insightful examples written as `e.g. This is my example`

We should check this for all inputs.